### PR TITLE
feat(rabbitmq-connector): add documentation for rabbitMq connector

### PR DESCRIPTION
**Pull request by Oleksiivanov**
## What is the purpose of the change

Add rabbitmq connector documentation

## Are there related marketing activities

https://github.com/camunda/connector-rabbitmq/issues/7

## When should this change go live?

15th November

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+id: rabbitmq
+title: RabbitMq Service Connector
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:3]:
```suggestion
title: RabbitMQ Service Connector
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+sidebar_label: RabbitMq Connector
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:4]:
```suggestion
sidebar_label: RabbitMQ Connector
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+description: Send messages to RabbitMq from your BPMN process.
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:5]:
```suggestion
description: Send messages to RabbitMQ from your BPMN process.
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+---
+
+The RabbitMq Connector allows you to connect your BPMN service with [RabbitMq](https://www.rabbitmq.com/).
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:8]:
```suggestion
**The RabbitMQ Connector** allows you to connect your BPMN service with [RabbitMQ](https://www.rabbitmq.com/).
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+## Prerequisites
+
+To use the **RabbitMq Connector**, you need to have an installed RabbitMq server and credentials for use it.
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:12]:
```suggestion
To use the **RabbitMQ Connector**, you need to have an installed RabbitMQ server and credentials to use it.
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+It is highly recommended not to expose your secret credentials as plain text but rather use Camunda secrets. See an [appendix entry](#how-do-i-store-secrets-for-my-connector) to learn more.
+Make sure that you write correct exchange name and routing key, because **RabbitMQ Connector** can't throw exception if it is wrong. 
+
+
+## Create an RabbitMq Connector task
+
+To use the **RabbitMq Connector** in your process, either change the type of existing task by clicking on it and using the wrench-shaped **Change type** context menu icon, or create a new Connector task by using the **Append Connector** context menu. Follow our [guide on using Connectors](../use-connectors.md) to learn more.
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:19]:
```suggestion
To use the **RabbitMQ Connector** in your process, either change the type of existing task by clicking on it and using the wrench-shaped **Change type** context menu icon, or create a new Connector task by using the **Append Connector** context menu. Follow our [guide on using Connectors](../use-connectors.md) to learn more.
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+The RabbitMq Connector allows you to connect your BPMN service with [RabbitMq](https://www.rabbitmq.com/).
+
+## Prerequisites
+
+To use the **RabbitMq Connector**, you need to have an installed RabbitMq server and credentials for use it.
+It is highly recommended not to expose your secret credentials as plain text but rather use Camunda secrets. See an [appendix entry](#how-do-i-store-secrets-for-my-connector) to learn more.
+Make sure that you write correct exchange name and routing key, because **RabbitMQ Connector** can't throw exception if it is wrong. 
+
+
+## Create an RabbitMq Connector task
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:17]:
```suggestion
## Create a RabbitMQ Connector task
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+To use the **RabbitMq Connector** in your process, either change the type of existing task by clicking on it and using the wrench-shaped **Change type** context menu icon, or create a new Connector task by using the **Append Connector** context menu. Follow our [guide on using Connectors](../use-connectors.md) to learn more.
+
+## Make your RabbitMq Connector for sending messages executable
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:21]:
```suggestion
## Make your RabbitMQ Connector for sending messages executable
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+
+To make the **RabbitMq Connector** executable, choose needed connection type in **Authentication** section and fill out the mandatory fields highlighted in red in the properties panel:
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:23]:
```suggestion
To make the **RabbitMQ Connector** executable, choose needed connection type in **Authentication** section and fill out the mandatory fields highlighted in red in the properties panel:
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+
+![connectors-rabbitmq-red-properties](../img/connectors-rabbitmq-red-properties.png)
+
+:::note
+All the mandatory and non-mandatory fields will be covered in the upcoming sections. Depending on the authentication selection you make, more fields might be required. We will also cover this in the next section.
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:28]:
```suggestion
All the mandatory and non-mandatory fields will be covered in the upcoming sections. Depending on the authentication selection you choose, different amount of fields are required. We will also cover this in the next section.
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+:::
+
+## Authentication
+
+You can choose among the available REST Connectors according to your authentication requirements.
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:33]: Sorry, what REST connectors?

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+First of all you need create new user in rabbitMq with needed permissions see more in [rabbitmq access control specification](https://www.rabbitmq.com/access-control.html).
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:34]:
```suggestion
First of all, you need have a user in your RabbitMQ instance with necessary permissions. See more at the [RabbitMQ access control specification](https://www.rabbitmq.com/access-control.html).
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+In next step choose need type of connection:
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:35]:
```suggestion
In next step choose a required type of connection:
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+
+### URI type connection 
+
+1. Click **URI** connection type in **Authentication** section
+2. Set **URI** to `URI`. It must contain rabbitMq username, password, host name, port number, and virtual host. Example: `amqp://userName:password@serverHost:port/virtualHost`, follow [uri rabbitmq specification](https://www.rabbitmq.com/uri-spec.html) to learn more.
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:40]:
```suggestion
2. Set **URI** to `URI`. It must contain RabbitMQ username, password, host name, port number, and virtual host. Example: `amqp://userName:password@serverHost:port/virtualHost`, follow the [RabbitMQ URI specification](https://www.rabbitmq.com/uri-spec.html) to learn more.
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+
+In **Routing** section you need set routing data (free feel use json format and FEEL expression format) :
+
+```json
+{
+  "exchange": "your-rabbitmq-exchange-name",
+  "routingKey": "your-rabbitmq-routing-key-name",
+  "virtualHost": "your-rabbitmq-virtual-host-name",
+  "hostName": "your-rabbitmq-host-name-or-ip-address",
+  "port": "your-rabbitmq-port-number"
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:59]:
```suggestion
{
  "exchange": "<your-rabbitmq-exchange-name>",
  "routingKey": "<your-rabbitmq-routing-key-name>",
  "virtualHost": "<your-rabbitmq-virtual-host-name>",
  "hostName": "<your-rabbitmq-host-name-or-ip-address>",
  "port": "<your-rabbitmq-port-number>"
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 16]
```diff
@@ -0,0 +1,97 @@
...
+}
+```
+
+For **URI** type connection required fields : `exchange` and `exchangeDemo`.
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:63]: What is `exchangeDemo`?

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 17]
```diff
@@ -0,0 +1,97 @@
...
+For **Credentials** type connection all fields is required. 
+
+![connectors-rabbitmq-routing](../img/connectors-rabbitmq-routing.png)
+
+
+## Message
+
+1. Set your message payload to **Message** section. Message can be text and json format.
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:71]:
```suggestion
1. Set your message payload to **Message** section. Message can be text and JSON format.
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 18]
```diff
@@ -0,0 +1,97 @@
...
+2. (Optional) Set message properties (json format or FEEL expression format) to **Properties** section. Follow [rabbitmq documentation](https://www.rabbitmq.com/publishers.html#message-properties) for learn more about message properties.
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:72]:
```suggestion
2. (Optional) Set message properties (JSON format or FEEL expression format) to **Properties** section. Follow [RabbitMQ documentation](https://www.rabbitmq.com/publishers.html#message-properties) for learn more about message properties.
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 19]
```diff
@@ -0,0 +1,97 @@
...
+
+![connectors-rabbitmq-message-with-properties](../img/connectors-rabbitmq-message-with-properties.png)
+
+
+## RabbitMq Connector response
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:77]:
```suggestion
## RabbitMQ Connector response
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 20]
```diff
@@ -0,0 +1,97 @@
...
+
+The **RabbitMq Connector** returns the Success result.
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:79]:
```suggestion
The **RabbitMQ Connector** returns the `Success` result.
```

**Comment by igpetrov** [comment]
@MoAtCamunda this is RabbitMQ documentation

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 21]
```diff
@@ -0,0 +1,103 @@
...
+  "exchange": "<your-rabbitmq-exchange-name>",
+  "routingKey": "<your-rabbitmq-routing-key-name>",
+  "virtualHost": "<your-rabbitmq-virtual-host-name>",
+  "hostName": "<your-rabbitmq-host-name-or-ip-address>",
+  "port": "<your-rabbitmq-port-number>"
+}
+```
+
+For a **URI** type connection, the required fields are `exchange` and `routingKey`.
+For a **Credentials** type connection, all fields are required.
```
Comment (christinaausley) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:72]: To confirm, filling out all of the fields is required here?

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 22]
```diff
@@ -0,0 +1,103 @@
...
+
+![connectors-rabbitmq-routing](../img/connectors-rabbitmq-routing.png)
+
+## Message
+
+1. Set your message payload to **Message** section. Message can be text and JSON format.
```
Comment (christinaausley) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:78]: What do you mean here? Do you mean, "In the **Message** section, insert the message payload."?

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 23]
```diff
@@ -0,0 +1,103 @@
...
+2. (Optional) Set message properties (JSON format or FEEL expression format) to **Properties** section. Follow [RabbitMQ documentation](https://www.rabbitmq.com/publishers.html#message-properties) for learn more about message properties.
```
Comment (christinaausley) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:79]: Similar comment here. Do you mean "In the **Properties** section, insert the message properties in JSON or FEEL expression format."?

**Review by christinaausley — APPROVED** [review]
A few non-blocking comments for clarification and technical adjustments in my latest commit if you'd like to take a closer look. Is there a reason why this PR is still a draft? Are you planning on adding changes, or can this be moved out of draft?

**Comment by christinaausley** [comment]
@Oleksiivanov No problem, just wanted to double check I reviewed all of your planned changes. If you're more comfortable keeping this as a draft until you're ready for prod, feel free to do so 👍

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md (hunk 21 above)
Comment (christinaausley) [reply docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:72]: Added in my latest commit 👍

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md (hunk 22 above)
Comment (christinaausley) [reply docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:78]: Added in my latest commit.

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md (hunk 23 above)
Comment (christinaausley) [reply docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:79]: Added in my latest commit.

**Comment by igpetrov** [comment]
@MoAtCamunda could you please take a look at it?

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 24]
```diff
@@ -0,0 +1,103 @@
...
+title: RabbitMQ Service Connector
+sidebar_label: RabbitMQ Connector
+description: Send messages to RabbitMQ from your BPMN process using the RabbitMQ Connector.
+---
+
+The **RabbitMQ Connector** allows you to connect your BPMN service with [RabbitMQ](https://www.rabbitmq.com/).
+
+## Prerequisites
+
+To use the **RabbitMQ Connector**, you need to have an installed RabbitMQ server and credentials to use it.
```
Comment (MoAtCamunda) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:12]:
```suggestion
To use the **RabbitMQ Connector**, you need to have an installed RabbitMQ server and relevant credentials.
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 25]
```diff
@@ -0,0 +1,103 @@
...
+It is highly recommended not to expose your secret credentials as plain text, but rather use Camunda secrets. See [this appendix entry](#how-do-i-store-secrets-for-my-connector) to learn more.
+
+:::note
+Ensure you write out the correct exchange name and routing key, because the **RabbitMQ Connector** can't throw an exception if they are incorrect.
```
Comment (MoAtCamunda) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:16]:
```suggestion
Ensure you write out the correct exchange name and routing key, as the **RabbitMQ Connector** can't throw an exception if they are incorrect.
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 26]
```diff
@@ -0,0 +1,103 @@
...
+:::
+
+## Create a RabbitMQ Connector task
+
+To use the **RabbitMQ Connector** in your process, either change the type of existing task by clicking on it and using the wrench-shaped **Change type** context menu icon, or create a new Connector task by using the **Append Connector** context menu. Follow our [guide to using Connectors](../use-connectors.md) to learn more.
+
+## Make your RabbitMQ Connector for sending messages executable
+
+To make the **RabbitMQ Connector** executable, choose the needed connection type in the **Authentication** section and fill out the mandatory fields highlighted in red in the properties panel:
```
Comment (MoAtCamunda) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:25]:
```suggestion
To connect to RabbitMQ, choose the required connection type in the **Authentication** section and complete the mandatory fields highlighted in red in the properties panel:
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 27]
```diff
@@ -0,0 +1,103 @@
...
+
+:::note
+Ensure you write out the correct exchange name and routing key, because the **RabbitMQ Connector** can't throw an exception if they are incorrect.
+:::
+
+## Create a RabbitMQ Connector task
+
+To use the **RabbitMQ Connector** in your process, either change the type of existing task by clicking on it and using the wrench-shaped **Change type** context menu icon, or create a new Connector task by using the **Append Connector** context menu. Follow our [guide to using Connectors](../use-connectors.md) to learn more.
+
+## Make your RabbitMQ Connector for sending messages executable
```
Comment (MoAtCamunda) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:23]:
```suggestion
## Connecting to RabbitMQ and sending messages
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 28]
```diff
@@ -0,0 +1,103 @@
...
+For a URI connection, take the following steps:
+
+1. Click the **URI** connection type in the **Authentication** section
+2. Set **URI** to `URI`. It must contain RabbitMQ username, password, host name, port number, and virtual host. For example, `amqp://userName:password@serverHost:port/virtualHost`; follow the [RabbitMQ URI specification](https://www.rabbitmq.com/uri-spec.html) to learn more.
+
+![connectors-rabbitmq-uri-fields](../img/connectors-rabbitmq-uri-fields.png)
+
+### Credentials type connection
+
+For a credentials connection, take the following steps:
```
Comment (MoAtCamunda) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:51]:
```suggestion
To connect with credentials, take the following steps:
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 29]
```diff
@@ -0,0 +1,103 @@
...
+
+1. Click the **Username/Password** connection type in the **Authentication** section
+2. Set the **Password** to `Password`.
+   ![connectors-rabbitmq-credentials-fields](../img/connectors-rabbitmq-credentials-fields.png)
+
+## Routing data
+
+In the **Routing** section you must set routing data (free feel use JSON format and FEEL expression format):
```
Comment (MoAtCamunda) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:59]:
```suggestion
In the **Routing** section you must set the routing data attributes (you can either use JSON or a FEEL expression):
```

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 30]
```diff
@@ -0,0 +1,103 @@
...
+
+```json
+{
+  "exchange": "<your-rabbitmq-exchange-name>",
+  "routingKey": "<your-rabbitmq-routing-key-name>",
+  "virtualHost": "<your-rabbitmq-virtual-host-name>",
+  "hostName": "<your-rabbitmq-host-name-or-ip-address>",
+  "port": "<your-rabbitmq-port-number>"
+}
+```
```
Comment (MoAtCamunda) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:69]: Can we add a FEEL expression example here?

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 31]
```diff
@@ -0,0 +1,103 @@
...
+
+For a **URI** type connection, the required fields are `exchange` and `routingKey`.
+For a **Credentials** type connection, the required fields are `exchange`, `routingKey`, `virtualHost`, `hostName`, and `port`.
+
+![connectors-rabbitmq-routing](../img/connectors-rabbitmq-routing.png)
+
+## Message
+
+1. In the **Message** section, insert the message payload. The message can be text and JSON format.
+2. (Optional) In the **Properties** section, insert the message properties in JSON or FEEL expression format. Follow [RabbitMQ documentation](https://www.rabbitmq.com/publishers.html#message-properties) for learn more about message properties.
```
Comment (MoAtCamunda) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:79]:
```suggestion
2. (Optional) In the **Properties** section, insert the message properties in JSON or as a FEEL expression. Follow [RabbitMQ documentation](https://www.rabbitmq.com/publishers.html#message-properties) for learn more about message properties.
```

**Review by MoAtCamunda — CHANGES_REQUESTED** [review]

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md (hunk 30 above)
Comment (MoAtCamunda) [reply docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:70]: excellent! thanks

**Review by MoAtCamunda — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]

File: versioned_docs/version-8.0/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 32]
```diff
@@ -0,0 +1,117 @@
...
+
+The **RabbitMQ Connector** allows you to connect your BPMN service with [RabbitMQ](https://www.rabbitmq.com/).
+
+## Prerequisites
+
+To use the **RabbitMQ Connector**, you need to have an installed RabbitMQ server and relevant credentials.
+It is highly recommended not to expose your secret credentials as plain text, but rather use Camunda secrets. See [this appendix entry](#how-do-i-store-secrets-for-my-connector) to learn more.
+
+:::note
+Ensure you write out the correct exchange name and routing key, as the **RabbitMQ Connector** can't throw an exception if they are incorrect.
```
Comment (MoAtCamunda) [inline versioned_docs/version-8.0/components/connectors/out-of-the-box-connectors/rabbitmq.md:16]:
```suggestion
Make sure the exchange name and routing key is correct, as the **RabbitMQ Connector** can't throw an exception if they are invalid.
```

File: versioned_docs/version-8.0/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 33]
```diff
@@ -0,0 +1,117 @@
...
+:::
+
+## Create a RabbitMQ Connector task
+
+To use the **RabbitMQ Connector** in your process, either change the type of existing task by clicking on it and using the wrench-shaped **Change type** context menu icon, or create a new Connector task by using the **Append Connector** context menu. Follow our [guide to using Connectors](../use-connectors.md) to learn more.
```
Comment (MoAtCamunda) [inline versioned_docs/version-8.0/components/connectors/out-of-the-box-connectors/rabbitmq.md:21]:
```suggestion
To use the **RabbitMQ Connector** in your process, either change the type of the existing task by clicking on it and using the wrench-shaped **Change type** context menu icon, or create a new Connector task by using the **Append Connector** context menu. Follow our [guide to using Connectors](../use-connectors.md) to learn more.
```

File: versioned_docs/version-8.0/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 34]
```diff
@@ -0,0 +1,117 @@
...
+First, you must have a user in your RabbitMQ instance with the necessary permissions. See more at the [RabbitMQ access control specification](https://www.rabbitmq.com/access-control.html).
+
+Next, we will choose a type of connection.
+
+### URI type connection
+
+For a URI connection, take the following steps:
+
+1. Click the **URI** connection type in the **Authentication** section
+2. Set **URI** to `URI`. It must contain RabbitMQ username, password, host name, port number, and virtual host. For example, `amqp://userName:password@serverHost:port/virtualHost`; follow the [RabbitMQ URI specification](https://www.rabbitmq.com/uri-spec.html) to learn more.
```
Comment (MoAtCamunda) [inline versioned_docs/version-8.0/components/connectors/out-of-the-box-connectors/rabbitmq.md:45]:
```suggestion
2. Set **URI** to `URI`. It must contain the RabbitMQ username, password, host name, port number, and virtual host. For example, `amqp://userName:password@serverHost:port/virtualHost`; follow the [RabbitMQ URI specification](https://www.rabbitmq.com/uri-spec.html) to learn more.
```

**Review by MoAtCamunda — CHANGES_REQUESTED** [review]
added a few minor adjustments

**Review by igpetrov — DISMISSED** [review]

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 35]
```diff
@@ -0,0 +1,117 @@
...
+
+## RabbitMQ Connector response
+
+The **RabbitMQ Connector** returns the `Success` result.
+The response contains a `messageId` variable.
+
+You can use an output mapping to map the response:
+
+1. Use **Result Variable** to store the response in a process variable. For example, `myResultVariable`.
+2. Use **Result Expression** to map fields from the response into process variables. For example:
```
Comment (MoAtCamunda) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:105]:
```suggestion
2. Use **Result Expression** to map specific fields from the response into process variables using [FEEL](https://docs.camunda.io/docs/components/modeler/feel/what-is-feel/). For example:
```

**Review by MoAtCamunda — CHANGES_REQUESTED** [review]

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 36]
```diff
@@ -0,0 +1,117 @@
...
+To use the **RabbitMQ Connector**, you need to have an installed RabbitMQ server and relevant credentials.
+It is highly recommended not to expose your secret credentials as plain text, but rather use Camunda secrets. See [this appendix entry](#how-do-i-store-secrets-for-my-connector) to learn more.
+
+:::note
+Ensure you write out the correct exchange name and routing key, as the **RabbitMQ Connector** can't throw an exception if they are incorrect.
+:::
+
+## Create a RabbitMQ Connector task
+
+To use the **RabbitMQ Connector** in your process, either change the type of existing task by clicking on it and using the wrench-shaped **Change type** context menu icon, or create a new Connector task by using the **Append Connector** context menu. Follow our [guide to using Connectors](../use-connectors.md) to learn more.
```
[the remark this answers is not in the document]
Comment (igpetrov) [reply docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:21]: @Oleksiivanov can you please create a follow up ticket to add icons as a separate task?

**Review by igpetrov — APPROVED** [review]

**Review by igpetrov — DISMISSED** [review]
LGTM. Re-approving after changes.

**Review by MoAtCamunda — APPROVED** [review]

File: versioned_docs/version-8.0/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 37]
```diff
@@ -0,0 +1,125 @@
...
+
+![connectors-rabbitmq-red-properties](../img/connectors-rabbitmq-red-properties.png)
+
+:::note
+All the mandatory and non-mandatory fields depending on the authentication selection you choose are covered in the upcoming sections.
+:::
+
+## Authentication
+
+You can choose among the available RabbitMQ Connectors according to your authentication requirements.
```
Comment (mbiciin) [inline versioned_docs/version-8.0/components/connectors/out-of-the-box-connectors/rabbitmq.md:43]: Are you referring here to the different authentication types? If that's the case, I would suggest something like:
"You can choose from the available authentication types of the RabbitMQ Connector depending on your authentication requirements."

**Review by mbiciin — COMMENTED** [review]
Looks very good. :)

**Review by igpetrov — APPROVED** [review]

**Pull request merged**